queue_services/entity-filer/src/entity_filer/worker.py

```diff
@@ -180,6 +180,13 @@ async def process_filing(filing_msg: Dict, flask_app: Flask):  # pylint: disable
 
                 elif filing.get('dissolution'):
                     dissolution.process(business, filing, filing_submission, filing_meta)
+                    if business.legal_type in ('SP', 'GP'):
```

Comment:
We may not need this if condition. HISTORICAL state is applicable for all legal type I think! @lekshmimallika-aot can you confirm?

Reply:
Fixed